redundant-bmc/src/sibling.hpp

```diff
@@ -145,13 +146,24 @@ class Sibling
     void clearCallbacks(Role role)
     {
         redEnabledCBs.erase(role);
+        bmcStateCBs.erase(role);
```

Comment:
Never a fan of the same thing in two places (in case one changes and we forget the other). Can we just have this call the new clearBMCStateCallback() function?

Reply:
Fixed.  Also added a clearHeartbeatCallback() that works the same way.